### ISSUE TITLE
Vision loading all files in the helper's directory

### ISSUE THREAD

**Issue by donatasbacius** [hapijs/vision#92]
This causes warnings to be logged to stdout. Could we change this to load only specific files?

Atm, if we have js.map files or tests or something else in that same folder, it will spit out warnings every app start.

**Comment by jagoda** [comment]
These warnings are the result of the discussion in #73. My concern with only loading specific files is that there is no good way to know the full set of file name patterns that are currently being used across all consumers. For example, you'll note that in #24 a user mentions using `.es6` rather than `.js` for the helper extension. While I think that logging warning messages should probably continue to be the default behavior (based on the discussion in #73), I think it would be reasonable to add an option to suppress these warnings if they are unwanted. Thoughts?

**Comment by lostthetrail** [comment]
Here is what comes to my mind.

**Option 1**
Add an `ignoreHelperRegistrationWarnings` flag.

```javascript
{
    ignoreHelperRegistrationWarnings: true
}
```

**Option 2**
Add `helperFilePattern` which could use something like glob for pattern support.

```javascript
{
    helperFilePattern: *.(js|es6)
}
```

**Option 3**
Add `helperExtension` which could be pretty dumb and catch many use-cases.

```javascript
{
    helperExtension: js
}
```

**Comment by wswoodruff** [comment]
@donatasbacius @lostthetrail This should be fixed by #119
